build_scripts/windows/scripts/build.cmd

```diff
@@ -122,7 +122,6 @@ set PYTHON_EXE=%PYTHON_DIR%\python.exe
 robocopy %PYTHON_DIR% %BUILDING_DIR% /s /NFL /NDL
 
 set CLI_SRC=%REPO_ROOT%\src
-%BUILDING_DIR%\python.exe -m pip install --no-warn-script-location --force-reinstall pycparser==2.18
```

Comment:
`pycparser==2.18` does not work. It is in the requirements.txt and version is `2.19`.

Reply:
`pycparser==2.18` was added by https://github.com/Azure/azure-cli/pull/7662. Funny thing is that no one knows why it was introduced in the first place.